src/GraphQL.php

```diff
@@ -503,10 +503,41 @@ protected function getSchemaConfiguration($schema)
 
         $schema = is_array($schema) ? $schema : $this->schemas[$schemaName];
 
-        if (! is_string($schema)) {
+        return static::getNormalizedSchemaConfiguration($schema);
+    }
+
+    /**
+     * @return array<string, array|Schema>
+     */
+    public static function getNormalizedSchemasConfiguration(): array
+    {
+        return array_filter(array_map(function ($schema) {
+            try {
+                return static::getNormalizedSchemaConfiguration($schema);
+            } catch (SchemaNotFound $e) {
```

Comment:
@jasonvarga do you remember why you added this here?

I was playing around with some refactoring in other parts when I saw this and I wondered: if a schema is broken, aka not found, why would I silence the schema here?

Aka: a broken schema throws an exception for a purpose, it should crash the app?

Reply:
Good question. 😬 

In 6.2.0, if you had an invalid class (or just a string, really, since class names weren't supported yet) it would not throw an exception. I think I just made it continue to _not_ throw an exception to keep the behavior consistent and not a breaking change.

Reply:
Thanks!

I decided to change it (https://github.com/rebing/graphql-laravel/pull/766) because somehow it didn't make sense to have an invalid configuration being silently skipped (as this is used for generated the routes, you're suddenly left in the dark why a certain schema simply "isn't there").

Reply:
No arguments here!